src/main/kotlin/no/skatteetaten/aurora/cantus/controller/NexusController.kt

```diff
@@ -22,4 +32,84 @@ class NexusController(val nexusService: NexusService) {
         val snapshotVersions = nexusService.getAllVersions(imageGroup, name, SNAPSHOT_REPO)
         return Flux.concat(releaseVersions, snapshotVersions)
     }
+
+    @PostMapping("/image/move")
+    fun moveImage(
+        @RequestBody moveImageCmd: MoveImageCommand
+        // TODO: add code to verify Flyttebil as source
+    ): Mono<MoveImageResult> {
+        // Search for image and validate that it correspond with exactly one instance in the expected repo
+        if (moveImageCmd.sha256.isEmpty()) return Mono.just(
+            MoveImageResult(
```

Comment:
Burde vi returnere client error i dette tilfellet? 
Kan vi bruke javax.validation.Valid for å validere?

Reply:
Det kan helt sikkert bruke noe som er spesielt laget for validering, ja.